rdkit/Chem/ChemicalFeatures.py

```diff
@@ -8,8 +7,8 @@
 #  which is included in the file license.txt, found at the root
 #  of the RDKit source tree.
 #
-from rdkit.Chem.rdMolChemicalFeatures import *
 from rdkit.Chem.rdChemicalFeatures import *
```

Comment:
That would be bad indeed (and clearly an indication why wildcard import is bad). I checked the imports now and there is no overlap between the two imports. Same for the other file.